tests/plugins/widget/manual/clipboardhtml.md

```diff
@@ -0,0 +1,34 @@
+@bender-tags: 4.13.0, feature, 3138
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, undo, wysiwygarea, toolbar
+
+**NOTE:** It's expected that if you paste the same copied content multiple times, widget copy counter won't change. Counter changes only during `copy`, `cut` and `drop` operations. Please also note that the counter is the same for every widget instance.
+
+1. Manipulate the content by:
+
+	* copying and pasting
+	* cutting and pasting
+	* dragging and droping
```

Comment:
Not sure why this happens:

![image](https://user-images.githubusercontent.com/1061942/64530884-72c49700-d30e-11e9-976c-cac6424cd780.png)

but there is something wrong with formatting :P